variables.tf

```diff
@@ -53,11 +64,7 @@ variable "options" {
   default     = {}
 }
 
-variable "network_config" {
-  description = "Network configuration options as defined in https://www.terraform.io/docs/providers/google/r/data_fusion_instance.html#network_config. Setting this implies a private instance."
-  type = object({
-    network       = string
-    ip_allocation = string
-  })
-  default = null
+variable "data_fusion_service_account" {
```

Comment:
I'm not an expert on Data Fusion, but does this need to be a variable or can we derive it in the module?

Reply:
it is an output of `google_data_fusion_instance` (which is not part of this module but meant to be used in conjunction with this module).

Reply:
Got it.